packages/webapp/src/components/CreateUserAccount/index.jsx

```diff
@@ -105,6 +105,9 @@ export default function PureCreateUserAccount({ onSignUp, email, onGoBack, isNot
         label={t('CREATE_USER.FULL_NAME')}
         placeholder={'e.g. Juan Perez'}
         hookFormRegister={register(NAME, { required: true })}
+        onBlur={(e) => {
+          e.target.value = e.target.value.trimStart();
```

Comment:
seems to be working well for me -- any reason we don't want to just use `.trim()` to prevent trailing spaces as well?